### PR TITLE
Model change for rt film feed localized title

### PR DESCRIPTION
**Pull request by AdamDM**

### CODE REVIEW

File: src/main/java/org/atlasapi/persistence/media/entity/PersonTranslator.java [hunk 1]
```diff
@@ -23,6 +24,13 @@
     private static final String BIRTH_PLACE_KEY = "birthPlace";
     private static final String QUOTES_KEY = "quotes";
     private static final String AWARDS = "awards";
+
+    private static final String PSEUDO_FORENAME = "pseudoForname";
+    private static final String PSEUDO_SURNAME = "pseudoSurname";
+    private static final String ADDITIONAL_INFORMATION = "additionalInformation";
+    private static final String BILLING = "billing";
+    private static final String SOURCE = "source";
+    private static final String SOURCE_TITLE = "sourceTitle";
```
Comment (jamiesprax) [inline src/main/java/org/atlasapi/persistence/media/entity/PersonTranslator.java]: minor thing, the whitespace on ln:27 & also assuming these are also keys, add _KEY on the end to follow the previous convention?

File: src/main/java/org/atlasapi/persistence/media/entity/ReviewTranslator.java [hunk 2]
```diff
@@ -1,25 +1,63 @@
...
+                TranslatorUtils.toString(dbObject, REVIEW_KEY));
+        review.setType(TranslatorUtils.toString(dbObject, TYPE_KEY));
+
+        List<DBObject> crewMembers = TranslatorUtils.toDBObjectList(dbObject, PEOPLE_KEY);
+        List<CrewMember> crewMemberList = new ArrayList<CrewMember>();
+
+        for (DBObject crewMember : crewMembers) {
+            crewMemberList.add(crewMemberTranslator.fromDBObject(crewMember, new CrewMember()));
+        }
+        review.setPeople(crewMemberList);
```
Comment (jamiesprax) [inline src/main/java/org/atlasapi/persistence/media/entity/ReviewTranslator.java]: Should be able to use a stream and collect to immutable list here

File: src/main/java/org/atlasapi/persistence/media/entity/DescribedTranslator.java [hunk 3]
```diff
@@ -402,6 +434,18 @@ public DBObject apply(Review review) {
             
                 });
     }
+
+    private void encodeDistribution(DBObject dbObject, Described entity) {
+        TranslatorUtils.fromIterable(dbObject, DISTRIBUTION_KEY, entity.getDistributions(),
+                new Function<Distribution, DBObject>() {
```
Comment (jamiesprax) [inline src/main/java/org/atlasapi/persistence/media/entity/DescribedTranslator.java]: lambda

File: src/main/java/org/atlasapi/persistence/media/entity/DescribedTranslator.java [hunk 4]
```diff
@@ -230,6 +246,17 @@ public LocalizedTitle apply(DBObject input) {
                 }));
     }
 
+    private void decodeDistribution(DBObject dbObject, Described entity) {
+        entity.setDistributions(TranslatorUtils.toIterable(dbObject,
+                DISTRIBUTION_KEY, new Function<DBObject, Distribution>() {
```
Comment (jamiesprax) [inline src/main/java/org/atlasapi/persistence/media/entity/DescribedTranslator.java]: lambda

File: src/main/java/org/atlasapi/persistence/media/entity/ReviewTranslator.java [hunk 5]
```diff
@@ -1,25 +1,71 @@
...
         TranslatorUtils.from(dbObject, REVIEW_KEY, model.getReview());
+        TranslatorUtils.from(dbObject, TYPE_KEY, model.getType());
+        TranslatorUtils.from(dbObject, PEOPLE_KEY, model.getPeople());
+
+        List<DBObject> peopleDbList = model.getPeople().stream()
+                .map(person -> personTranslator.toDBObject(dbObject, person))
+                .collect(Collectors.toList());
+
+        BasicDBList dbPeople = new BasicDBList();
+        for (DBObject dbObject1 : peopleDbList) {
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/persistence/media/entity/ReviewTranslator.java]: Variable naming: `dbPerson` is more descriptive

**Pull request closed without merging**